ERROR - 2024-08-04 17:08:57 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\projects\template-to-codeigniter-4\system\Database\Database.php(68): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\projects\template-to-codeigniter-4\system\Database\Config.php(111): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\projects\template-to-codeigniter-4\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Config::forge()
#5 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#6 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:create', Array)
#7 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#8 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#9 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\projects\template-to-codeigniter-4\system\Database\Database.php(68): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\projects\template-to-codeigniter-4\system\Database\Config.php(111): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\projects\template-to-codeigniter-4\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Config::forge()
#4 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#5 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:create', Array)
#6 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#7 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#8 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#9 {main}
ERROR - 2024-08-04 17:36:58 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\projects\template-to-codeigniter-4\system\Database\Database.php(68): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\projects\template-to-codeigniter-4\system\Database\Config.php(111): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\projects\template-to-codeigniter-4\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Config::forge()
#5 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#6 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:create', Array)
#7 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#8 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#9 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\projects\template-to-codeigniter-4\system\Database\Database.php(68): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\projects\template-to-codeigniter-4\system\Database\Config.php(111): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\projects\template-to-codeigniter-4\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Config::forge()
#4 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#5 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:create', Array)
#6 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#7 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#8 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#9 {main}
ERROR - 2024-08-04 18:05:47 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\projects\template-to-codeigniter-4\system\Database\Database.php(68): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\projects\template-to-codeigniter-4\system\Database\Config.php(111): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 C:\projects\template-to-codeigniter-4\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Config::forge()
#5 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#6 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:create', Array)
#7 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#8 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#9 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\projects\template-to-codeigniter-4\system\Database\Database.php(68): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\projects\template-to-codeigniter-4\system\Database\Config.php(111): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 C:\projects\template-to-codeigniter-4\system\Commands\Database\CreateDatabase.php(136): CodeIgniter\Database\Config::forge()
#4 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#5 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:create', Array)
#6 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#7 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#8 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#9 {main}
ERROR - 2024-08-04 19:00:11 --> mysqli_sql_exception: Incorrect database name '' in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php(312): mysqli->query('SHOW TABLES FRO...', 0)
#1 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SHOW TABLES FRO...')
#2 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SHOW TABLES FRO...')
#3 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\projects\template-to-codeigniter-4\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\projects\template-to-codeigniter-4\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\projects\template-to-codeigniter-4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-08-04 19:51:45 --> mysqli_sql_exception: Unknown table 'blog_ci.posts' in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php(312): mysqli->query('DROP TABLE `pos...', 0)
#1 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `pos...')
#2 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `pos...')
#3 C:\projects\template-to-codeigniter-4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `pos...')
#4 C:\projects\template-to-codeigniter-4\app\Database\Migrations\2024-08-04-180754_Post.php(67): CodeIgniter\Database\Forge->dropTable('posts')
#5 C:\projects\template-to-codeigniter-4\system\Database\MigrationRunner.php(867): App\Database\Migrations\Post->down()
#6 C:\projects\template-to-codeigniter-4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\projects\template-to-codeigniter-4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\projects\template-to-codeigniter-4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\projects\template-to-codeigniter-4\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2024-08-04 20:04:11 --> mysqli_sql_exception: Failed to open the referenced table 'users' in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\projects\template-to-codeigniter-4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\projects\template-to-codeigniter-4\app\Database\Migrations\2024-08-04-180754_Post.php(62): CodeIgniter\Database\Forge->createTable('posts')
#5 C:\projects\template-to-codeigniter-4\system\Database\MigrationRunner.php(867): App\Database\Migrations\Post->up()
#6 C:\projects\template-to-codeigniter-4\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\projects\template-to-codeigniter-4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\projects\template-to-codeigniter-4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\projects\template-to-codeigniter-4\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2024-08-04 20:05:40 --> mysqli_sql_exception: Unknown table 'blog_ci.users' in C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\projects\template-to-codeigniter-4\system\Database\MySQLi\Connection.php(312): mysqli->query('DROP TABLE `use...', 0)
#1 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `use...')
#2 C:\projects\template-to-codeigniter-4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `use...')
#3 C:\projects\template-to-codeigniter-4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `use...')
#4 C:\projects\template-to-codeigniter-4\app\Database\Migrations\2024-08-04-180739_User.php(57): CodeIgniter\Database\Forge->dropTable('users')
#5 C:\projects\template-to-codeigniter-4\system\Database\MigrationRunner.php(867): App\Database\Migrations\User->down()
#6 C:\projects\template-to-codeigniter-4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\projects\template-to-codeigniter-4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\projects\template-to-codeigniter-4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\projects\template-to-codeigniter-4\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\projects\template-to-codeigniter-4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\projects\template-to-codeigniter-4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\projects\template-to-codeigniter-4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\projects\template-to-codeigniter-4\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\projects\template-to-codeigniter-4\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
